src/main/java/org/opensearch/timeseries/ml/MemoryAwareConcurrentHashmap.java

```diff
@@ -55,48 +52,4 @@ public ModelState<RCFModelType> put(String key, ModelState<RCFModelType> value)
         }
         return previousAssociatedState;
     }
-
-    /**
-     * Gets all of a config's model sizes hosted on a node
-     *
-     * @param configId config Id
-     * @return a map of model id to its memory size
-     */
-    public Map<String, Long> getModelSize(String configId) {
```

Comment:
where these just never used anymore?

Reply:
yeah, they are not used after recent refactoring.